tutorials/Example_Track_on_Radar_Segment_on_Satellite/track_on_radar_segment_on_satellite.ipynb

```diff
@@ -0,0 +1,1245 @@
+       "}\n",
+       "\n",
+       ".xr-attrs,\n",
+       ".xr-var-attrs,\n",
+       ".xr-var-data,\n",
+       ".xr-index-data {\n",
+       "  grid-column: 1 / -1;\n",
+       "}\n",
+       "\n",
+       "dl.xr-attrs {\n",
+       "  padding: 0;\n",
+       "  margin: 0;\n",
+       "  display: grid;\n",
+       "  grid-template-columns: 125px auto;\n",
+       "}\n",
+       "\n",
+       ".xr-attrs dt,\n",
+       ".xr-attrs dd {\n",
+       "  padding: 0;\n",
+       "  margin: 0;\n",
+       "  float: left;\n",
+       "  padding-right: 10px;\n",
+       "  width: auto;\n",
+       "}\n",
+       "\n",
+       ".xr-attrs dt {\n",
+       "  font-weight: normal;\n",
+       "  grid-column: 1;\n",
+       "}\n",
+       "\n",
+       ".xr-attrs dt:hover span {\n",
+       "  display: inline-block;\n",
+       "  background: var(--xr-background-color);\n",
+       "  padding-right: 10px;\n",
+       "}\n",
+       "\n",
+       ".xr-attrs dd {\n",
+       "  grid-column: 2;\n",
+       "  white-space: pre-wrap;\n",
+       "  word-break: break-all;\n",
+       "}\n",
+       "\n",
+       ".xr-icon-database,\n",
+       ".xr-icon-file-text2,\n",
+       ".xr-no-icon {\n",
+       "  display: inline-block;\n",
+       "  vertical-align: middle;\n",
+       "  width: 1em;\n",
+       "  height: 1.5em !important;\n",
+       "  stroke-width: 0;\n",
+       "  stroke: currentColor;\n",
+       "  fill: currentColor;\n",
+       "}\n",
+       "</style><pre class='xr-text-repr-fallback'>&lt;xarray.Dataset&gt;\n",
+       "Dimensions:    (time: 1, Y: 1500, X: 2500)\n",
+       "Coordinates:\n",
+       "  * time       (time) datetime64[ns] 2021-05-26T15:56:15.477525888\n",
+       "    longitude  (Y, X) float64 nan nan nan nan ... -61.97 -61.95 -61.93 -61.91\n",
+       "    latitude   (Y, X) float64 nan nan nan nan nan ... 14.64 14.64 14.64 14.64\n",
+       "Dimensions without coordinates: Y, X\n",
+       "Data variables:\n",
+       "    C10        (time, Y, X) float32 nan nan nan nan ... 264.4 264.4 264.5 264.7</pre><div class='xr-wrap' style='display:none'><div class='xr-header'><div class='xr-obj-type'>xarray.Dataset</div></div><ul class='xr-sections'><li class='xr-section-item'><input id='section-33dbb822-d16b-4d26-a5f2-08fc8fb8bbce' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-33dbb822-d16b-4d26-a5f2-08fc8fb8bbce' class='xr-section-summary'  title='Expand/collapse section'>Dimensions:</label><div class='xr-section-inline-details'><ul class='xr-dim-list'><li><span class='xr-has-index'>time</span>: 1</li><li><span>Y</span>: 1500</li><li><span>X</span>: 2500</li></ul></div><div class='xr-section-details'></div></li><li class='xr-section-item'><input id='section-7bd1d3fa-4051-4ced-bf34-8aaa537ca26f' class='xr-section-summary-in' type='checkbox'  checked><label for='section-7bd1d3fa-4051-4ced-bf34-8aaa537ca26f' class='xr-section-summary' >Coordinates: <span>(3)</span></label><div clas
```

Comment:
This is a bit of a hack fix as it assumes all the features are at the first time step, which isn't necessarily the case for larger datasets. It could be better to pull the times from `goes_adj_features` rather than `in_goes_for_tobac`